strawberryfields/backends/tfbackend/ops.py

```diff
@@ -113,7 +113,7 @@ def get_prefac_tensor(D, directory, save):
 def squeezed_vacuum_vector(r, theta, D, batched=False, eps=1e-32):
     """returns the ket representing a single mode squeezed vacuum state"""
     if batched:
-        batch_size = r.shape[0].value
+        batch_size = r.shape[0]
```

Comment:
Nice that it works this way now :+1:

Reply:
The call to .numpy() made the tests pass, so I assumed it was the required behaviour.
If we want a different behaviour then we should revise the tests first.
But I agree with the consideration that measurement results are by definition not trainable.

Reply:
I have pushed the numpy call to the tests (only when the tensors are tf.Tensor objects). It works well, so I think problem solved 👍